Link to comparison operators page (#11026)
trinodb/trino: docs/src/main/sphinx/sql/show-catalogs.rst

Description
-----------

List the available catalogs. 

> :ref:`Specify a pattern <like_operator>` in the optional `LIKE` clause to filter

Reviewer: Would suggest making "`LIKE` clause" the link text, as the subject of the link

Reviewer 2: I'm not following Joe's suggestion here and throughout. Jessie, did you figure out what he meant? Can I see a sample of the reworked sentence?

Reviewer 3: he suggests to make the phrase "LIKE clause" the link to the other page .. I would leave it

Reviewer: The destination of this link talks about the LIKE clause, not about specifying patterns in general, so my preference is to be specific about what info the reader will get when they click.

But it's a really nitpicky point in this case, so I think it's fine to leave as well. Just offering some clarity on what I meant.

the results to the desired subset. For example, the following query allows you

> to find catalogs where the first character in a value is specified::

Reviewer: "that start with the `t` character::"

Reviewer 3: Add this at the end of the last sentence to detail what the example actually does..  and same for others below

Description
-----------

List the available catalogs. 

> :ref:`Specifying a pattern <like_operator>` allows you to filter the results to

Reviewer 4: Let's use consistent terminology. It's not a "pattern" it's the `LIKE clause` (which accepts a pattern).

nit: like_operator should be called `like_clause` IMO.

Author: Thanks for the correction.

Regarding consistency in labeling, I see LIKE being referred to as both an operator and clause on the internet. Within our docs, though, the word operator is used to reference to LIKE, BETWEEN, IS NULL, and others, already.

Reviewer 3: We could use

Specifying a pattern in the optional LIKE clause  .. that would be correct and consistent with the code snippet where `pattern` is used..

-----------

List the available catalogs.

:ref:`Specify a pattern <like_operator>` in the optional ``LIKE`` clause to

> filter the results to the desired subset. For example, the following query
> allows you to find catalogs where the first character in a value is specified.
> Query results show catalogs that begin with ``t``::

Reviewer 4: ```suggestion
filter the results to the desired subset. For example, the following query
allows you to find catalogs whose names begin with ``t``::
```

Similar in other places too? Existing wording seems repetitive (first we describe what query does in very abstract terms and then follow with the concrete meaning).

Reviewer 3: I like that .. want to adapt to that @Jessie212 ?

Author: Great suggestion, @hashhar. Applied.
